remoteFortisoar/info.json

```diff
@@ -0,0 +1,242 @@
+{
+    "name": "remoteFortisoar",
+    "version": "1.0.0",
+    "label": "Remote FortiSOAR",
+    "description": "This connector allows you to run REST API operations on other FortiSOAR environments",
+    "publisher": "Fortinet",
+    "icon_small_name": "fortisoar_remote_small.png",
+    "icon_large_name": "fortisoar_remote_large.png",
+    "cs_approved": true,
+    "category": "Utilities",
+    "help_online": "",
+    "configuration": {
+        "fields": [
+            {
+                "title": "FortiSOAR Endpoint URL",
+                "type": "text",
+                "name": "url",
+                "required": true,
+                "visible": true,
+                "editable": true
+            },
+            {
+                "title": "Authentication Type",
+                "name": "auth_type",
+                "required": true,
+                "editable": true,
+                "visible": true,
+                "type": "select",
+                "options": [
+                    "Basic",
+                    "HMAC"
+                ],
+                "value": "Basic",
+                "description": "Authentication that will be used to connect and communicate with the FortiSOAR server. Choose either Basic or HMAC. By default, this is set to Basic.",
+                "onchange": {
+                    "Basic": [
+                        {
+                            "description": "User name used to authenticate FortiSOAR Server",
+                            "title": "User Name",
+                            "tooltip": "User name used authenticate FortiSOAR Server",
+                            "name": "username",
+                            "type": "text",
+                            "visible": true,
+                            "editable": true,
+                            "required": true,
+                            "value": ""
+                        },
+                        {
+                            "description": "Password used to authenticate FortiSOAR Server",
+                            "title": "Password",
+                            "tooltip": "Password used to authenticate FortiSOAR Server",
+                            "required": true,
+                            "editable": true,
+                            "visible": false,
```

Comment:
true. not allowing to configure basic auth currently

Reply:
Done